### PR TITLE
FIX: Corrected a typo in HTML docs

### PR DESCRIPTION
**Pull request by souravsingh**

### CODE REVIEW

File: doc/manual/source_localization/forward.rst [hunk 1]
```diff
@@ -483,7 +483,7 @@ in Neuromag software. The coil types fall in two general categories:
 - Axial gradiometers and planar gradiometers
   and
 
-- Planar gradiometers.
+- Planar magnetometers.
```
Comment (agramfort) [inline doc/manual/source_localization/forward.rst]: this was correct

**Comment by souravsingh** [comment]
The fix was related to Issue #2800 about an HTML Typo

**Comment by souravsingh** [comment]
I am sorry if I got something wrong

File: doc/manual/source_localization/forward.rst (hunk 1 above)
Comment (agramfort) [inline doc/manual/source_localization/forward.rst]: my bad. You're right

**Pull request merged**